src/scripts/main.js

```diff
@@ -1,3 +1,16 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+
+[...table.rows].forEach(item => {
+  item.insertCell(4).innerHTML = item.cells[1].innerText;
+});
+
+const header = document.createElement('th');
+const footer = document.createElement('th');
+
+header.innerText = table.tHead.rows[0].cells[1].innerText;
+footer.innerText = table.tFoot.rows[0].cells[1].innerText;
+
+table.tHead.rows[0].cells[4].replaceWith(header);
+table.tFoot.rows[0].cells[4].replaceWith(footer);
```

Comment:
```suggestion
```

Here you are repeating yourself. Actually, you don't need this part of code. Everything is done in your `forEach`. You can delete these lines and you will see, that it works ALMOST the same, as before.
So I would recommend you to clear these lines and use `cloneNode` in your `forEach` instead of copying just innerHtml (and something like `before` instead of `insertCell`)

Reply:
Thanks for the tips! I used your method and I improved the task solution)